src/libra_py/cube_file_methods.py

```diff
@@ -260,3 +240,137 @@ def integrate_cube(cube_A, cube_B, grid_volume):
 
 
 
+def plot_cubes( params ):
+    """
+    This function plots the cubes for selected energy levels using VMD.
+    
+    Args:
+    
+        params (dict):
+    
+        ks_orbital_indicies (list): This list contains the Kohn-Sham orbital indices. Using this list we can
+                                    generate the minimum state and obtain the index of the states to be plotted.
+        states_to_be_plotted (list): The list containing the Kohn-Sham orbitals to be plotted by VMD. This list is defined in the submit file.
+        path_to_tcl_file (str): The path to the tcl file which contains the input for plotting the cubes in VMD.
+        MO_images_directory (str): The molecular orbitals images directory.
+    
+        isUKS (int): This parameter is set for spin restricted and unrestricted calculations. When it is
+                     set to 1 it means that unrestricted calculations were set in the input file otherwise 
+                     it is restricted.
+        curr_step (int): The current time step used to save the images of the MOs.
+        phase_factor_visual (numpy array): The phase correction factor list for each MOs for the current step.
+    
+    """

```

Comment:
please add the handling of the input dictionary to ensure the input provided has all the critical parameters and if some non-critical ones are not provided - set  them up to the default values. 
Again, see the example in the dynamics/tsh/compute/run_dynamics